### PR TITLE
General README.md

### PR DESCRIPTION
**Pull request by BowTiedHeron**
Still needs an update for standard coding practises.

### CODE REVIEW

**Comment by gonwi** [comment]
HI,

It's standard practice to add a CONTRIBUTING.md file where you explain how to create PRs, issues, code style and testing.

We should also have a CODE_OF_CONDUCT.md file.

And a .github folder with templates for Issues and PRs

I can help with this when I am free

File: README.md [hunk 1]
```diff
@@ -1,34 +1,57 @@
...
+- [GemFarm](https://github.com/gemworks/gem-farm) for Staking as a Service through a service layer API built by Honey Finance - WIP
+- [Honey-SDK](https://github.com/honey-labs/honey-sdk) for permissionless lending pools that use NFTs as collateral - WIP
+
 ## Getting Started
 
-First, run the development server:
+First things first,
+
+Make sure to create a `.env` file based on the `.env.example` file. The RPC_URL is the standard mainnet RPC.
+You should replace this with a custom one if you are in need of better performance.
```
Comment (BowTiedFirefox) [inline README.md:14]: add "you might not need it if you are development on devnet but currently we do not have farm available on devnet, However we will add instructions to add farm to devnet in the future"

File: README.md [hunk 2]
```diff
@@ -1,34 +1,57 @@
 This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
 
+## General overview of what this repo contains
+
+This repo contains a frontend interface built with nextJS, the UI connects to:
+- [GemFarm](https://github.com/gemworks/gem-farm) for Staking as a Service through a service layer API built by Honey Finance - WIP
+- [Honey-SDK](https://github.com/honey-labs/honey-sdk) for permissionless lending pools that use NFTs as collateral - WIP
```
Comment (BowTiedFirefox) [inline README.md:7]: add "the documentation about Goki wallet https://github.com/GokiProtocol/walletkit
and degen-ui https://degen-xyz.vercel.app/ "

File: README.md (hunk 1 above)
Comment (BowTiedHeron) [reply README.md:14]: good addition! will add it tomorrow.

File: README.md (hunk 2 above)
Comment (BowTiedHeron) [reply README.md:7]: good addition! will add it tomorrow.

**Review by tinybigideas — COMMENTED** [review]
Dropping a review as someone who is using these docs: All looks good. To add completeness, I'd want a PR template  too, especially if we should be linking to the issue we're working on (and we should probably create that issue first?). Use `closes #xyz` etc

**Comment by BowTiedHeron** [comment]
pushed an initial version to development in collaboration with pyro
because of time constraints I had to copy it instead of merging
this should not be standard practice, next time we will do this different

**Comment by gonwi** [comment]
I can help here

**Comment by BowTiedHeron** [comment]
initiated a PR template file in the .github folder on the dev branch
everything should be on the dev branch at this moment
new dev cycle starts next week, I suggest to branch off from development for a fresh start
copied all the notes from above and I will close this PR and delete the readme branch

**Pull request closed without merging**